src/util/linkify.js

```diff
@@ -0,0 +1,28 @@
+
+/**
+ * Replaces URLs in a block of text with HTML anchor tags.
+ *
+ * @method linkify
+ * @param {String} text The text on which to operate
+ * @param {Object} options optional modifiers
+ * @param {Boolean} [options.target] what to set as the target attribute
+ * @return {String} The modified text.
+ */
+export default function linkify(text, options) {
+	if (!text) return '';
+	options = options || {};
+
+	const urlRegex = /((((ht|f){1}(tps?:[/][/]){1}))[-a-zA-Z0-9@:;%_\+.~#?&/=]+)/gm,
+		createLink = (href) => {
+			const target = options.target || '',
+				targetAttr = `target="${target}"`;
+
+			if (target === '_blank') {
+				targetAttr += ' rel="noopener noreferrer"';
+			}
```

Comment:
in order to avoid the Travis error and be more accurate with the var name, this block could be

```js
const relAttr = target === '_blank' ? 'rel="noopener noreferrer"' : '';

return `<a href="${href}" title="${href}" ${targetAttr} ${relAttr}>${href}</a>`;
```